### ISSUE TITLE
Code mark animations cause box-shadow problems on examples.

### ISSUE THREAD

**Issue by dabrahams** [stlab/better-code#2]
Currently, because assets/js/reveal-mark-plugin/style.css has

```css
.reveal div.mark-fragments [data-mark].fragment pre {
    box-shadow: none;
}
```

it drops the left and right shadows when the first mark animation happens, because the overlaid `div.fragment` at `absolute` position has left/right margins that make it obscure the sides of the box shadow behind.  But without this clause the box shadows for the overlapping fragments accumulate and get darker with each step.

The current animation strategy is based on the reveal.js highlight plugin, which makes the initial state a non-[fragment](https://revealjs.com/fragments/) and then overlays fragments for the following steps, and to make this whole thing a non-hack we may need to rethink that approach… but probably someone with CSS-fu could figure out a smaller workaround.

**Comment by dabrahams** [comment]
Actually my workaround doesn't seem to work.  It seems to be somewhat random whether these shadows accumulate as I develop, but after a clean reboot, they are accumulating even with the workaround in place

**Comment by dabrahams** [comment]
Fixed by 382ce6b8c
